superset/config.py

```diff
@@ -1420,6 +1421,17 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
     "port": internet_port,
 }
 
+# By default, the Welcome page features example charts and dashboards. This can be
+# changed to show all charts/dashboards the user has access to, or a custom view
+# by providing the title and a FAB filter:
+# WELCOME_PAGE_LAST_TAB = (
+#     "Xyz",
+#     [{"col": 'created_by', "opr": 'rel_o_m', "value": 10}],
+# )
+WELCOME_PAGE_LAST_TAB: Union[
+    Literal["examples", "all"], Tuple[str, List[Dict[str, Any]]]
```

Comment:
Should we use more strict typing for FAB filter?

Reply:
Good thinking! I actually already tried it, but I wasn't able to find a type for the Filter in the current FAB codebase (typing work is still ongoing). Once the type is added upstream I'll try to update these `Dict[str, Any]` to proper `TypedDict`